Makefile

```diff
@@ -1,17 +1,22 @@
-IMAGE_NAME=secrets-store-csi-driver-provider-azure
+-include secrets.env
+export $(shell test -f secrets.env && sed 's/=.*//' secrets.env)
+
 REGISTRY_NAME ?= upstreamk8sci
 REGISTRY ?= $(REGISTRY_NAME).azurecr.io
 DOCKER_IMAGE ?= $(REGISTRY)/public/k8s/csi/secrets-store/provider-azure
 IMAGE_VERSION ?= 0.0.4
+IMAGE_NAME ?= secrets-store-csi-driver-provider-azure
+
 # Use a custom version for E2E tests if we are testing in CI
 ifdef CI
 override IMAGE_VERSION := e2e-$$(git rev-parse --short HEAD)
 endif
+
+
 BUILD_DATE=$$(date +%Y-%m-%d-%H:%M)
 GO_FILES=$(shell go list ./...)
 ORG_PATH=github.com/Azure
-PROJECT_NAME := secrets-store-csi-driver-provider-azure
```

Comment:
I removed the `PROJECT_NAME` since this is the same as `IMAGE_NAME` . And replaced where necessary